app/assets/stylesheets/refills/_navigation.scss

```diff
@@ -0,0 +1,96 @@
+header.navigation {
+  $navigation-background: $blue;
+  $navigation-color: transparentize(white, 0.3);
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.